src/packages/admin-ui-components/src/detail-panel/graphql.ts

```diff
@@ -39,3 +39,17 @@ export const getRelationshipQuery = (entityName: string, summaryField?: string)
     }
   `;
 };
+
+export const getUploadUrlMutation = gql`
+	mutation GetUploadUrl($key: ID!) {
+		getUploadUrl(key: $key)
+	}
+`;
+
+export const createSubmissionMutation = gql`
+	mutation Mutation($createSubmissionData: SubmissionInsertInput!) {
+		createSubmission(data: $createSubmissionData) {
```

Comment:
Submission is example specific so this needs to be generic also.

Reply:
Will follow pattern for getRelationshipQuery to dynamically generate.

Reply:
This is actually leftovers, this mutation is not needed, handled by createEntity mutation.